app-ios/tutanota/Sources/Contacts/ContactsSynchronization.swift

```diff
@@ -24,10 +24,28 @@ private let ALL_SUPPORTED_CONTACT_KEYS: [CNKeyDescriptor] = [
 class ContactsSynchronization {
   init(userPreferencesProvider: UserPreferencesProvider) {
     self.userPreferencesProvider = userPreferencesProvider
+    self.queryLocalContainer()
   }
-  
+
   let userPreferencesProvider: UserPreferencesProvider
-  
+  var localContainer: String = CNContactStore().defaultContainerIdentifier()
```

Comment:
this doesn't need to be a variable, we should just assign one or another in the beginning (or you can make it a lazy or computed variable)